streams/src/test/java/org/apache/kafka/streams/KafkaStreamsTest.java

```diff
@@ -192,6 +192,10 @@ private void prepareStreams() throws Exception {
         ClientMetrics.addApplicationIdMetric(anyObject(StreamsMetricsImpl.class), EasyMock.eq(APPLICATION_ID));
         ClientMetrics.addTopologyDescriptionMetric(anyObject(StreamsMetricsImpl.class), anyString());
         ClientMetrics.addStateMetric(anyObject(StreamsMetricsImpl.class), anyObject());
+        ClientMetrics.addNumAliveStreamThreadMetric(anyObject(StreamsMetricsImpl.class), anyObject());
+        EasyMock.expectLastCall().anyTimes();
```

Comment:
Actually this is added intentionally since there are three tests in which we only created the instance without calling its `start()` function -- thinking about it, I will move the registration logic back to constructor and then we can remove it.